frb_rust/src/thread_pool/web.rs

```diff
@@ -1,4 +1,38 @@
-pub use crate::third_party::wasm_bindgen::worker_pool::WorkerPool as SimpleThreadPool;
+use crate::third_party::wasm_bindgen::worker_pool::WorkerPool;
```

Comment:
nit: is it possible we still use the "pub use ... as ...", then everything in web.rs can be kept unchanged